docs/validator/staking-overview.md

```diff
@@ -27,4 +27,5 @@ The protocol automatically elects validators by issuing an auction for each epoc
     * [Economics in Sharded Blockchain](https://nearprotocol.com/papers/economics-in-sharded-blockchain/) to know more about the incentives structure of NEAR.
     * [Sharding Design: Nightshade](https://nearprotocol.com/papers/nightshade) to gain a more detailed understanding of our consensus mechanism.
 
-4. If anything is unclear, you get stuck, or you just want to chat with fellow validators, please head over to our [Telegram group](https://t.me/near_validators) or [Discord](https://discord.gg/ZMPr3VB). There we have a dedicated validator channel to answer all of your questions.
+4. If anything is unclear or you get stuck, please head over to our official chat on [Slack](https://near.chat) in the channels `#community-validator-announcement` and `#community-validator-troubleshooting`.
```

Comment:
We are likely going to connect our Slack channel with the Telegram and Discord. Thus, please leave those links and do not link to the Slack. I will keep you informed should anything change.